### PR TITLE
Adding flowPosition support to dataflow. Upgraded HPA to autoscaling/v2

### PR DESCRIPTION
**Pull request by r65535**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?

- autoscaling/v2 has now been released. Migrated the helm template to deploy v2 instead of v2beta2
- NiFiKop's NiFiDataflow CRD supports flowPosition. Added support for this in helm
- Upped version of chart to `v1.1.0`

### Why?

To support flowPosition via helm

### Checklist

- [x] Implementation tested
- [x] Error handling code meets the [guideline](docs/error-handling-guide.md)
- [x] Logging code meets the guideline
- [x] User guide and development docs updated (if needed)
- [x] Append changelog with changes

### CODE REVIEW

File: helm/nifi-cluster/templates/nifi-dataflow.yaml [hunk 1]
```diff
@@ -8,6 +8,11 @@ spec:
   bucketId: {{ .bucketId }}
   flowId: {{ .flowId }}
   flowVersion: {{ .flowVersion }}
+ {{ if .flowPosition }}
```
Comment (mh013370) [inline helm/nifi-cluster/templates/nifi-dataflow.yaml:11]: Nice catch!

File: helm/nifi-cluster/Chart.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.1.0
```
Comment (mh013370) [inline helm/nifi-cluster/Chart.yaml:33]: I think this was missed in the release process. @juldrixx : Do you know how/where versions are updated when a release occurs? 

I don't see that releases are created via a circleCI job

**Comment by mh013370** [comment]
LGTM

**Review by mh013370 — APPROVED** [review]

File: helm/nifi-cluster/Chart.yaml (hunk 2 above)
Comment (juldrixx) [reply helm/nifi-cluster/Chart.yaml:33]: They are created manually. And it should be 1.1.1 not 1.1.0

Comment (r65535) [reply helm/nifi-cluster/Chart.yaml]: Apologies, I was versioning the helm chart separately to the overall nifikop release 😄 

I've pushed another commit to change this over to 1.1.1

**Review by juldrixx — APPROVED** [review]

**Pull request merged**